dev_guide/routes.adoc

```diff
@@ -185,6 +197,16 @@ spec:
 ----
 ====
 
+For {product-title} Pro, use certificate and key files from a CA,then specify
```

Comment:
The long `oc create route edge` example command above (the one with the `--cert`, `--key`, and `--ca-cert` flags) and corresponding yaml block (the one with the `key:`, `certificate:`, and `caCertificate:`) can be included unconditionally now, and they would belong here.